src/lib.rs

```diff
@@ -219,6 +219,7 @@ pub async fn get_build_output(
 
     let mut subpackages = BTreeMap::new();
     let mut outputs = Vec::new();
+    let mut render_only_output = Vec::new();
```

Comment:
How about making this an array of `Output` and then print the JSON array at the end? With the current approach we are still printing 2 separate JSON strings which is not really _parseable_. What do you think?

Reply:
You mean at the end of this ?
```
for recipe_path in &recipe_paths {
                    let output =
                        get_build_output(&build_args, recipe_path.clone(), &log_handler).await?;
                    outputs.push(output);
                }
```
Yes, I think I miss looked it. Because if we pass multiple recipes, I guess we will have multiple arrays of json which aren't that parseable :)